content/src/fieldConfig/business-formation.json

```diff
@@ -250,7 +250,7 @@
       },
       "agentOfficeAddressMunicipality": {
         "label": "Office City",
-        "error": "Select a city."
+        "error": "Enter a city."
```

Comment:
Pebble - All other error states for registered agent contain the term agent, this does not. Designs should have been provided. I'd check with Content on what the content should be.

Reply:
Changed per Content's confirmation.